stablehlo/tests/chlo/legalize_chlo_decomposition.mlir

```diff
@@ -0,0 +1,533 @@
+// RUN: stablehlo-opt --pass-pipeline="builtin.module(func.func(stablehlo-legalize-chlo))" \
+// RUN:   --split-input-file --verify-diagnostics %s | FileCheck %s
+
+// CHECK-LABEL: func.func @asin_bf16(
```

Comment:
I'll add more test points once this is submitted, can copy some over from CHLO->MHLO. This is a good start.

Reply:
Created https://github.com/openxla/stablehlo/issues/1997 so I don't forget